packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dr-response.ts

```diff
@@ -150,27 +192,31 @@ async function handleSuccessResponse({
   }
 }
 
-export async function processDrResponseSoap({
-  drResponse: { response, success, gateway, outboundRequest },
+export async function processDrResponse({
+  drResponse: { errorResponse, mtomResponse, gateway, outboundRequest },
 }: {
   drResponse: DrSamlClientResponse;
 }): Promise<OutboundDocumentRetrievalResp> {
```

Comment:
Why an object param if it only contains one property? Couldn't it just be

```ts
function processDrResponse({ 
  errorResponse, mtomResponse, gateway, outboundRequest
}: DrSamlClientResponse)...
```

Reply:
This is when we have multiple parameters, not when there's only one. If we add more params, then we can update the function signature/contract.